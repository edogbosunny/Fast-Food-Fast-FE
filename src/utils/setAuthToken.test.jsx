import React from 'react';
import { shallow } from 'enzyme';
import SetAuthToken from './setAuthToken';

describe('Spinner Test', () => {
  const wrapper = shallow(<SetAuthToken />);

  it('should render spinner', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
