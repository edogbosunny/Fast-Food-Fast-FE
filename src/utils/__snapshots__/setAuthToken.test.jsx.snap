// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Test should render spinner 1`] = `""`;
